packages/blade/src/components/BottomSheet/__tests__/BottomSheet.ssr.test.tsx

```diff
@@ -11,7 +11,10 @@ beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
 afterAll(() => jest.restoreAllMocks());
 
 describe('<BottomSheet />', () => {
-  it('should render a BottomSheet ssr', () => {
+  // Skipping this test as it is fails in SSR with error: ReferenceError: navigator is not defined
+  // coming from @floating-ui/react, which is used in BottomSheet component
+  // https://github.com/floating-ui/floating-ui/blob/94ba5b7c7069c272c967e4eef6ba09a793880855/packages/utils/react/src/index.ts#L109
+  it.skip('should render a BottomSheet ssr', () => {
```

Comment:
is it specific to test or will bottomsheet break on actual SSR apps?

Reply:
I tried it in a Next.js app, seems to be working.

https://github.com/razorpay/blade/assets/46647141/a89084d9-5356-4a2e-8586-ff84cb7683a6